modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/portlet/ObjectEntriesPortlet.java

```diff
@@ -35,7 +35,7 @@ public ObjectEntriesPortlet(
 		ObjectActionLocalService objectActionLocalService,
 		long objectDefinitionId,
 		ObjectDefinitionLocalService objectDefinitionLocalService,
-		ObjectFieldFDSFilterFactoryRegistry objectFieldFDSFilterFactoryRegistry,
+		ObjectDefinitionDeployerImpl objectDefinitionDeployerImpl,
```

Comment:
Hey @Valmir-unicap, you should not use the Impl directly. It goes against the "Program to an interface, not an implementation" OOP principle.